applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -39,8 +39,13 @@ enum sw_codec_select {
 
 enum sw_codec_num_ch {
 	SW_CODEC_ZERO_CHANNELS,
```

Comment:
Would there be a condition use zero channel?
If yes, SW_CODEC_ZERO_CHANNEL (remove S) would be good.
If not, then probably we can use

```
enum sw_codec_num_ch {
	SW_CODEC_ONE_CHANNEL = 1,  /* Only use one channel */
	SW_CODEC_TWO_CHANNELS, /* Use both channels */
};
```

I was wondering if we could use number directly instead of enumeration, but since we use switch case to catch this in later, seems the enumeration is the best choice in current stage.